src/mimemail.erl

```diff
@@ -143,9 +144,19 @@ decode(OrigHeaders, Body, Options) ->
 					decode_component(Headers, Body, MimeVersion, Options);
 				{<<"multipart">>, _SubType, _Parameters} ->
 					erlang:error(non_mime_multipart);
-				{Type, SubType, Parameters} ->
+				{Type, SubType, CTParameters} ->
```

Comment:
the bug was here. We were returning the value of "content-type parameters" in place where we supposed to return `mimemail:parameters()` map.